python/lsst/sims/operations/Simulator.py

```diff
@@ -105,6 +108,8 @@ def __init__(self, lsstDB, obsProfile, sessionID, nRun, seeingEpoch, simStartDay
         self.telSeeing = telSeeing
         self.opticalDesSeeing = opticalDesSeeing
         self.cameraSeeing = cameraSeeing
+        self.scaleToNeff = scaleToNeff
+        self.atmNeffFactor = atmNeffFactor
```

Comment:
Why are the scaleToNeff and atmNeffFactor (and other parameters) stored here in the Simulator class, when they are also stored in the Filters class (of which there is one belonging to the Simulator class, as far as I can tell)?